test/xvfb-wrapper.h

```diff
@@ -28,17 +32,37 @@ struct test_function {
  * loop over in x11_tests_run() to extract the tests. This removes the
  * need of manually adding the tests to a suite or listing them somewhere.
  */
+#define CUSTOM_ELF_SECTION test_func_sec
```

Comment:
I'd prefer if this one was named `TEST_ELF_SECTION` or something instead of the more generic/ambiguous "custom".

Reply:
Sure, semantics matter!